MaxText/configs/models/kimi-k2-1t.yml

```diff
@@ -0,0 +1,53 @@
+# Copyright 2023–2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# model config for Kimi-k2 - 1T parameters
+# See https://huggingface.co/moonshotai/Kimi-K2-Instruct for more details.
+
+
+base_emb_dim: 7168
+base_num_query_heads: 64
+base_num_kv_heads: 64
+base_mlp_dim: 18432
+base_moe_mlp_dim: 2048
+base_num_decoder_layers: 61
+first_num_dense_layers: 1
+mlp_activations: ["silu", "linear"]
+vocab_size: 163840
+enable_dropout: False
+logits_via_embedding: False
+normalization_layer_epsilon: 1.0e-6
+num_experts: 384
+num_experts_per_tok: 8
+shared_experts: 1
+routed_scaling_factor: 2.827
+routed_score_func: "sigmoid"
+routed_bias: True
+decoder_block: "deepseek"
+# MLA
+attention_type: "mla"
+q_lora_rank: 1536
+kv_lora_rank: 512
+qk_nope_head_dim: 128
+qk_rope_head_dim: 64
+v_head_dim: 128
+mscale: 1.0
+# RoPE
+rope_type: "yarn"
+rope_max_timescale: 50000 # Kimi-k2 uses "rope_theta": 50000.0
```

Comment:
nit: `# Kimi-k2 uses "rope_theta": 50000.0` it seems config indicates that? 

Or we could just add "rope_theta" near `rope_max_timescale`?